content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -90,10 +90,10 @@ For example, this is how to start a simple web server as a static Pod:
     EOF
     ```
 
-3. Configure your kubelet on the node to use this directory by running it with `--pod-manifest-path=/etc/kubelet.d/` argument. On Fedora edit `/etc/kubernetes/kubelet` to include this line:
+3. Configure your kubelet on the node to use this directory by running it with `--pod-manifest-path=/etc/kubernetes/manifests/` argument. On Fedora edit `/etc/kubernetes/kubelet` to include this line:
```

Comment:
i'm not sure where the `/etc/kubernetes/kubelet` path is coming from and haven't seen it before in the k8s.io docs.

kubeadm uses different files for systemd / kubelet configuration:
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/kubelet-integration/#the-kubelet-drop-in-file-for-systemd

Reply:
I agree; even if it's wrong in the original, we should aim to fix this specific line.

Reply:
I am using a kind cluster and the `/etc/kubernetes/kubelet` path works for me on that environment.